### PR TITLE
Rewrite education benefits opt out test in Cypress, add helpers to fi…

### PR DESCRIPTION
**Pull request by CBonade**
…ll out form that mimic nightwatch helpers and test data required for this form

## Description
This test is a rewrite of the Nightwatch test for opting out of education benefits, with Cypress. 

## Testing done
Tested in Cypress runner with Chrome.  All tests seem to pass in my testing.

## Screenshots

image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/0993/tests/e2e/00.0993-opt-out.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      'not.contain',
+      '/claimant-information',
+    );
+
+    cy.get('label[name="privacyAgreementAccepted-label"]', {
+      timeout: Timeouts.slow,
+    })
+      .should('be.visible')
+      .then(() => {
+        cy.wait(Timeouts.normal);
```
Comment (ddzz) [inline src/applications/edu-benefits/0993/tests/e2e/00.0993-opt-out.cypress.spec.js:41]: Can we use something other than `cy.wait()` here?

Comment (CBonade) [reply src/applications/edu-benefits/0993/tests/e2e/00.0993-opt-out.cypress.spec.js]: Replaced this with adding the timeout to the `cy.get` on the following line instead

File: src/platform/testing/e2e/cypress/support/commands/form-helpers.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Cypress.Commands.add('overrideVetsGovApi', url => {
+  const current = window.VetsGov || {};
+  window.VetsGov = Object.assign({}, current, {
+    api: {
+      // eslint-disable-next-line object-shorthand
```
[the remark this answers is not in the document]
Comment (CBonade) [reply src/platform/testing/e2e/cypress/support/commands/form-helpers.js]: Removed

**Comment by CBonade** [comment]
Closing out this PR in lieu of a new one being created that includes these changes.

**Pull request closed without merging**